client/js/libs/handlebars/friendlydate.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+const moment = require("moment");
+
+module.exports = function(time) {
+	// See http://momentjs.com/docs/#/displaying/calendar-time/
+	return moment(new Date(time)).calendar(null, {
+		sameDay: "[Today]",
+		lastDay: "[Yesterday]",
+		lastWeek: "L", // Locale
```

Comment:
I feel you @YaManicKill.
I don't think a constant is a much better solution than a comment though (and I often find it's not a better one at all), I think we should just move on :)
Also, as we'll be adding more and more Moment.js stuff on the front-end, we'll probably have a common config piece for it so that will be more suitable there. But for later IMO (or else, for someone else 😅).